src/services/item/plugins/publication/published/service.ts

```diff
@@ -122,20 +131,39 @@ export class ItemPublishedService {
       return itemPublished;
     }
 
-    return await this.post(member, repositories, item, { canBePrivate: true });
+    return await this.post(member, repositories, item, publicationStatus, { canBePrivate: true });
+  }
+
+  private checkPublicationStatus({ id, type }: Item, publicationStatus: PublicationStatus) {
+    switch (publicationStatus) {
+      case PublicationStatus.ReadyToPublish:
+        return true;
+      case PublicationStatus.ItemTypeNotAllowed:
+        throw new ItemTypeNotAllowedToPublish(id, type);
+      case PublicationStatus.Published:
+      case PublicationStatus.PublishedChildren:
+        throw new ItemPublicationAlreadyExists(id);
+      case PublicationStatus.Unpublished:
+      case PublicationStatus.Pending:
+      case PublicationStatus.Invalid:
+      case PublicationStatus.Outdated:
+      case PublicationStatus.NotPublic:
```

Comment:
Is this removed with the update of SDK ?

Reply:
Yes, I have to remove it in this PR too ^^